From 474418761cea9080a856692a2653cc87934885cd Mon Sep 17 00:00:00 2001
From: Jinzhe Zeng <jinzhe.zeng@rutgers.edu>
Date: Thu, 10 Mar 2022 17:13:42 -0500
Subject: [PATCH 2/2] fix macos library name

The library type was changed from SHARED to MODULE in #1384.

Fixes errors in conda-forge/deepmd-kit-feedstock#31
---
 deepmd/env.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/deepmd/env.py b/deepmd/env.py
index 5942b4c0..422092b6 100644
--- a/deepmd/env.py
+++ b/deepmd/env.py
@@ -208,8 +208,8 @@ def get_module(module_name: str) -> "ModuleType":
     """
     if platform.system() == "Windows":
         ext = ".dll"
-    elif platform.system() == "Darwin":
-        ext = ".dylib"
+    #elif platform.system() == "Darwin":
+    #    ext = ".dylib"
     else:
         ext = ".so"
 
-- 
2.35.1

